include/stats.h

```diff
@@ -214,8 +214,8 @@ struct stats_hdr {
  * @param group__               The group containing the entry to clear.
  * @param var__                 The statistic entry to clear.
  */
-void stats_init(struct stats_hdr *shdr, uint8_t size, uint8_t cnt,
-		const struct stats_name_map *map, uint8_t map_cnt);
+void stats_init(struct stats_hdr *shdr, uint8_t size, uint16_t cnt,
```

Comment:
Shouldn't we have `u16_t` here?

Reply:
We probably should, but I wanted to keep existing conventions. I think that type alignment for stats.h should be done in a separate PR/commit

Reply:
@jukkar migration form stdint.h is out of scope his PR. Can we kept this as is?

Reply:
Well, it just looks weird that the header file has uint8_t and uint16_t vars, but the .c file uses u8_t and u16_t. In this respect I would say this needs to change.

Reply:
@jukkar, Stats API now uses Zephyr types